CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## [X.X.X]
+* Feature:
+	* Add function to update metadata in live
```

Comment:
"Add a public method to update now playing info metadata"

Reply:
done